NuRadioReco/modules/RNO_G/channelGlitchDetector.py

```diff
@@ -2,12 +2,7 @@
 from NuRadioReco.utilities import units
 from NuRadioReco.framework.parameters import channelParametersRNOG as chp
 
-
-
-import numpy as np
-import logging
-
-logger = logging.getLogger('NuRadioReco.RNO_G.channelGlitchDetector')
+import numpy as np, logging
```

Comment:
uh, I actually like this syntax but we use it nowhere and it is not pep8 conform I think